cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -212,7 +208,7 @@ func (suite *InstallerIntegrationTestSuite) TestInstallerOverwriteServiceApp() {
 	)
 	cp.Expect("Done")
 	cp.SendLine("exit")
-	cp.ExpectExitCode(0)
+	cp.ExpectExit() // the return code can vary depending on shell (e.g. zsh vs. bash); just assert the installer shell exited
```

Comment:
Will this type of check be necessary in other integration tests now?

It may be worth considering a new function that wraps `ExpectExitCode()` and `ExpectExit()` and uses either one depending on the current shell, but I will leave that up to your discretion.

Reply:
It could rear its head in other installer tests, but in all of our other subshell tests that use "exit" followed by "ExpectExitCode(0)", the tests pass. Nathan found a similar issue with shells on Windows: https://github.com/ActiveState/cli/commit/3d87ecbe06d8cea89561400762fc027ab59e5088. It's something we'll have to live with I guess.